subsys/net/lib/lwm2m/lwm2m_shell.c

```diff
@@ -642,7 +724,7 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 	SHELL_CMD_ARG(resume, NULL, LWM2M_HELP_RESUME, cmd_resume, 1, 0),
 	SHELL_CMD_ARG(lock, NULL, LWM2M_HELP_LOCK, cmd_lock, 1, 0),
 	SHELL_CMD_ARG(unlock, NULL, LWM2M_HELP_UNLOCK, cmd_unlock, 1, 0),
-
+	SHELL_CMD_ARG(observations, NULL, LWM2M_HELP_OBSERV, cmd_observations, 1, 0),
```

Comment:
```suggestion
	SHELL_CMD_ARG(obs, NULL, LWM2M_HELP_OBSERV, cmd_observations, 1, 0),
```

Is `observations` a bit long to type?
Maybe `obs` is enough.

Reply:
Done